README.md

```diff
@@ -1,15 +1,15 @@
-# README
+# Hangman Web Application Game
 
-This is a simple web application game of Hangman
+This is a simple web application game of Hangman.
 
 In order to run this game on your local system, keep these specs and steps in mind:
 
-* Ruby versions 2.7.0 or later to run Rails
```

Comment:
`bundle install` install all the dependencies (including rails), and that manage the specific versions of gems. So here you can say do `bundle install`